### ISSUE TITLE
ERROR:  Could not find the version of the CLI

### ISSUE THREAD

**Issue by jkonowitch** [kysely-org/kysely-ctl#5]
It appears the CLI is looking at the version of my application package (which is in a monorepo, using pnpm workspaces, and is not versioned) rather than the version of `kysely-ctl`.

```
 ERROR  Could not find the version of the CLI

  at getCLIInstalledVersion (/PROJECT/node_modules/.pnpm/kysely-ctl@0.4.0_c12@1.10.0_citty@0.1.6_consola@3.2.3_kysely@0.27.3_nypm@0.3.8_pathe@1.1.2_pk_2dkvxtov55pjiwaee5hspcr3bm/node_modules/kysely-ctl/dist/bin.js:77:11)
```

**Comment by igalklebanov** [comment]
Hey 👋

Try using `no-outdated-check` flag for now. Will be fixed ASAP.

**Comment by jkonowitch** [comment]
Ok! I just tried locally, and adding ` const { version } = await readPackageJSON(undefined, { startingFrom: import.meta.dirname})`; fixed the issue

https://github.com/kysely-org/kysely-ctl/blob/bf7d9947651e77a10aea8d080d6b39d83b2695d1/src/utils/version.mts#L15

**Comment by igalklebanov** [comment]
Yeah that should be the direction.
Just released 0.4.1, hope it fixes this. If not, I'd love a small reproduction repository.

**Comment by jkonowitch** [comment]
Yes, that fixed it 👍
